test/shared_test/lib_battery_test.cpp

```diff
@@ -63,7 +63,7 @@ TEST_F(lib_battery_thermal_test, updateTemperatureTestSubMinute) {
     }
     avgTemp /= 600.0;
     auto s = thermal_state({ 93.49, 16.86, 16.85 });
-    EXPECT_NEAR(avgTemp, s.T_batt, 0.02) << "updateTemperatureTest: 1";
```

Comment:
I'm not opposed to adding the "compare state" function here, but the reason I was using EXPECT_NEAR was to ensure that the average temperature during the sub-minute steps was the same as the average temperature during the hourly steps. Can we do both comparisons for more accurate coverage?

Reply:
Gotcha, I'll add that back in. Thanks